cacheops/conf.py

```diff
@@ -15,7 +15,8 @@ class Settings(object):
     CACHEOPS_DEFAULTS = {}
     CACHEOPS = {}
     CACHEOPS_LRU = False
-    CACHEOPS_DEGRADE_ON_FAILURE = False
+    CACHEOPS_DEGRADE_ON_FAILURE = True
+    CACHEOPS_DEGRADE_PERSISTENT_PER_REQUEST = False
```

Comment:
CACHEOPS_DEGRADE_WHOLE_REQUEST?
CACHEOPS_DEGRADE_TILL_REQUEST_FINISHED?
I read these settings like "Cacheops, honey, please, degrade on failure!" or "Cacheops, please, degrate till the end of request" :)
Or like a term: 
"persistent degrade for whole request".
I mean, proper english grammar is necessary here.